backend/homework/1/core/src/main/java/org/handson/Student.java

```diff
@@ -0,0 +1,43 @@
+package org.handson;
+
+public class Student
+{
+int id;
+String name;
+int age;
+char grade;
+public void setID(int ID)
```

Comment:
Make all variables as private and add a parameterised constructor + default constructor